src/pages/CompanyCourseManagement/CCMPage.jsx

```diff
@@ -0,0 +1,97 @@
+// Import necessary components from Material-UI library
+import Button from "@mui/material/Button";
+import { List, ListItem, ListItemText } from "@mui/material";
+// Import custom PageLayout component
+import PageLayout from "../../components/PageLayout";
+
+// CCMPage component definition
+function CCMPage() {
+  // Return JSX markup for the CCMPage component
+  return (
+    // Use the custom PageLayout component with specific title and viewport settings
+    <PageLayout title="CCM" viewportPage>
```

Comment:
Why is this a viewport Page? What happens, if you have 100 items?

Reply:
Viewport removed